src/components/ChatEntry.js

```diff
@@ -1,22 +1,40 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  // update the icon using props.liked
+  const likeButton = props.liked === true ? '❤️' : '🤍';
+
+  // determine local or remote sender
+  const senderType =
+    props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
```

Comment:
Tip: This is a little brittle since it means we couldn't re-use these components for a chat between other people.